settings/dconf-defaults/locks/00_settings

```diff
@@ -0,0 +1,8 @@
+# This file specifies gsettings keys which are immutable.
+# https://help.gnome.org/admin/system-admin-guide/stable/dconf-lockdown.html.en
+
+# EndlessOS users should not be able to enable or disable system-extensions.
+# This also have the side-effect that users won't be able to run their own extensions,
+# but we are still going on with that decision: https://phabricator.endlessm.com/T28777#801566
+/org/gnome/shell/enabled-extensions
+/org/gnome/shell/disabled-extensions
```

Comment:
You may also need to add the `/org/gnome/shell/disable-user-extensions` key here

Reply:
Good point.

Also, I noticed  that, one needs to set the key first before it can be locked down. From the [docs](https://help.gnome.org/admin/system-admin-guide/stable/dconf-lockdown.html.en):

> Before you can lock down a key or subpath, you need to set it. This example shows how to lock a background setting once it has been set.

Reply:
Have you seen https://help.gnome.org/admin/system-admin-guide/stable/extensions-lockdown.html.en as well?

Reply:
My suspicion is that having the override, and the `locks` file, is enough for what we want.

Reply:
I see. 

Updated the PR accordingly to https://help.gnome.org/admin/system-admin-guide/stable/extensions-lockdown.html.en